.github/workflows/pull-request.yml

```
@@ -3,19 +3,19 @@ name: Validate Pull Request
 on: pull_request
 
 env:
-  BALLERINA_DISTRIBUTION_VERSION: 2201.12.0   # Update this with the latest Ballerina version
+  BALLERINA_DISTRIBUTION_VERSION: 2201.14.0-20260527-050400-74f7e6bf   # Update this with the latest Ballerina version
 
 jobs:
   ubuntu-build:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)